apps/notifications/babel.config.js

```diff
@@ -0,0 +1,54 @@
+// @babel/preset-env modules option. `false` for ECMAScript modules.
+function modulesOption( opts ) {
+	if ( opts && opts.modules !== undefined ) {
+		return opts.modules;
+	}
+
+	if ( typeof process.env.MODULES !== 'undefined' ) {
+		return process.env.MODULES === 'esm' ? false : process.env.MODULES;
+	}
+
+	return false; // Default
+}
+
+module.exports = ( api, opts = {} ) => {
+	api.cache( false );
+
+	return {
+		presets: [
+			[
+				require.resolve( '@babel/preset-env' ),
+				{
+					corejs: opts.corejs ? opts.corejs : 3.6,
+					debug: opts.debug ? opts.debug : false,
+					bugfixes: opts.bugfixes ? opts.bugfixes : false,
+					modules: modulesOption( opts ),
+					useBuiltIns: opts.useBuiltIns ? opts.useBuiltIns : 'entry',
+					// Exclude transforms that make all code slower, see https://github.com/facebook/create-react-app/pull/5278
+					exclude: [ 'transform-typeof-symbol' ],
+				},
+			],
+			[
+				require.resolve( '@babel/preset-react' ),
+				{ runtime: 'automatic', importSource: 'preact/compat' },
```

Comment:
This ensures that the new JSX transform that no longer requires a `import React from 'react'` maps to Preact.